classes/itembase.class.php

```diff
@@ -850,6 +850,21 @@ public function item_mandatory_is_allowed() {
     }
 
     /**
+     * item_add_mandatory_base_fields
+     * Copy mandatory fields to $record.
+     *
+     * @param stdClass $record
+     * @return nothing
```

Comment:
here you can use `@return void` or no return phpdoc, as you prefer. But those `@return nothing` are not correct.